HTML/RESULTPAGE/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../CSS/General/style.css" />
+    <link rel="stylesheet" href="../../CSS/QUIZPAGE/style.css" />
+    <title>RESULT PAGE</title>
+  </head>
+  <body>
+    <main>
+      <h1>Result</h1>
+      <h3>Score : 23</h3>
+      <div class="question-list">
```

Comment:
Just a suggestion, since this is a kind of form input u can put the questions in a form and add a submit button as well , correct me if I'm wrong

Reply:
Thanks, Poornima for the feedback. Since It is just a screen So I have not made changes. But I'll do